### ISSUE TITLE
Detail View integration won't work if configured custom command uses other custom vars. 

### ISSUE THREAD

**Issue by mkayontour** [Icinga/icingaweb2-module-businessprocess#455]
TLDR; Custom vars may differ while using a custom command for the detail view integration.

## Problem
As stated in the documentation, you can use a custom command name for the detail view integration. 
[A clear and concise description of what the issue is.](https://github.com/Icinga/icingaweb2-module-businessprocess/blob/main/doc/10-Monitoring.md#detail-view-integration)

```
[...]
        $bpName = $service->customvars['icingacli_businessprocess_config'] ?? null;
[...]
        $nodeName = $service->customvars['icingacli_businessprocess_process'] ?? null;
[...]
```

https://github.com/Icinga/icingaweb2-module-businessprocess/blob/3403b605e4f08f5ff50c2ba183eac1aa9ca7b0f2/library/Businessprocess/ProvidedHook/Icingadb/ServiceDetailExtension.php#L49-L75

But with custom commands are **custom vars** specific for the command as well. As I'm sure many users have had custom commands of this because some started early to check processes before they were integrated in the ITL. 

## Expected behaviour

Have the possibility to allow own custom vars for the detail extension views. 

## Versions
Businessprocess: 2.5.1

**Comment by nilmerg** [comment]
> As I'm sure many users have had custom commands of this because some started early to check processes before they were integrated in the ITL.

I'm not so sure, [`icingacli_businessprocess_process` was the only option from the beginning](https://github.com/Icinga/icingaweb2-module-businessprocess/commit/bc54d5914663f7b8fbadcea37d62d366287c7751#diff-3f58042676cf25663a98860926c38db1961a43d11649f1fe317ec3e8a30d5a9bR64).

But it's certainly possible to support custom properties. Someday, or in case a volunteer takes over. :wink:

**Comment by mkayontour** [comment]
Thanks, I'll try to provide a PR. ;)
